src/ZendServerWebApi/Model/ZendServer.php

```diff
@@ -46,8 +46,9 @@ class ZendServer
             '8.0' => '1.9',
             '8.5' => '1.10',
             '9.0' => '1.12',
-    		'9.1' => '1.14',
+		'9.1' => '1.14',
```

Comment:
Please, open the file with very simple text editor. 

![image](https://user-images.githubusercontent.com/247452/55871575-3cca8d80-5b8b-11e9-82e1-a336a4731fdb.png)

The line with '9.1' needs to be aligned to the previous one. Also '2019.0' needs to be aligned. See the screenshot. Or check here:  https://github.com/zend-patterns/ZendServerWebApiModule/pull/18/files